x/crisis/internal/keeper/keeper.go

```diff
@@ -1,3 +1,4 @@
+// DONTCOVER
```

Comment:
I don't think we should add this to a keeper

Reply:
Agreed. Will remove and implement whatever tests I can!